src/Footer.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+const Footer = (props) => (
+  <ul>
+    {props.addressDetail.map((addressLine, index) => {
+      return (
+        <li key={index}>{addressLine}</li>
+      )
+    })}
```

Comment:
if the above changes are applied then the following would look something like this
```suggestion
         <li key={address}>{address}</li>
         <li key={email}>{email}</li>
         <li key={phoneNumber}>{phoneNumber}</li>
```

also I've replace the list key to be the values instead of an index. Generally React does not recommend using index as keys, but i know this has been covered in the course so far. Just something to keep in mind when working in React Projects outside of CYF

https://dev.to/shiv1998/why-not-to-use-index-as-key-in-react-lists-practical-example-3e66